ministryofjustice/correspondence_tool_staff
# CT-2425 Add new case tabbed navigation and hide others

## Description
Add new case tabs as per prototype: https://cts-prototype.herokuapp.com/cases
 
## Self-review checklist

 
### Screenshots
Old:
![image](https://user-images.githubusercontent.com/22935203/66743361-a34da280-ee71-11e9-81c6-e3e588569aed.png)

New:

![image](https://user-images.githubusercontent.com/22935203/66743265-6ed9e680-ee71-11e9-95b3-9be5ce48f230.png)
![image](https://user-images.githubusercontent.com/22935203/66743308-83b67a00-ee71-11e9-9565-44a220ce1007.png)

 
### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2425
 
### Deployment
n/a
 
### Manual testing instructions
View cases homepage and cycle through the new tabs, check highlight on global nav makes sense, and that the correct tab is highlighted, and nothing's broken
